src/link/style.scss

```diff
@@ -10,6 +10,51 @@
 	}
 }
 
+.wikipediapreview-edit-preview-popover {
+	.components-popover__content {
+		background: unset;
+		border: unset;
+		box-shadow: none;
+	}
+	.wikipediapreview {
+		box-shadow: none;
+	}
+	.wikipediapreview-body::after {
+		content: none;
+	}
+}
+
+.wikipediapreview-edit-preview-container {
+	display: flex;
+	align-items: center;
+}
+
+.wikipediapreview-edit-preview-controllers {
+	width: 52px;
+	height: 136px;
+	background-color: #000000;
+	display: flex;
+	flex-direction: column;
+	justify-content: space-around;
+	color: #FFFFFF;
+	border-top-right-radius: 4px;
+	border-bottom-right-radius: 4px;
+	padding: 3px;
+}
+
+// .wikipediapreview-edit-preview-controllers-change {
```

Comment:
you can use `&-change` under the controllers css, so you don't repeat all the prefix